app/controllers/api/v1/applications/sources/scenarios_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Api
+  module V1
+    module Applications
+      module Sources
+        class ScenariosController < Api::V1::Applications::Sources::ApplicationController
+          def index
+            scenario = event_source.scenarios.find(params[:id])
+
+            rows = Event.find_by(application: application, scenario: scenario)
+
+            response.headers["Access-Control-Allow-Origin"] = "*"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.